showpass/Showpass-E2E-tests
# TA-32-Profile - 'Email' form overall appearance

**Description:**
This spec will be verifying the following elements within the ‘Email’ form under user ‘Profile’.
- Captions
- Input fields
- Buttons

**Jira ticket:**
https://showpass.atlassian.net/browse/TA-32

**How to test:**
Run the following test suite and ensure the corresponding spec passes
cypress/e2e/user-settings/profile.cy.js

**Test execution results:**

<img width="1716" alt="TA-32-pass" src="https://github.com/showpass/Showpass-E2E-tests/assets/138615449/45ff46cc-5c35-4216-980f-685418fb921f">
